scripts/install/run_alkos.bash

```diff
@@ -9,7 +9,7 @@ source "${RUN_ALKOS_SCRIPT_SOURCE_DIR}/scripts/utils/pretty_print.bash"
 
 RUN_ALKOS_SCRIPT_QEMU_COMMAND="qemu-system-x86_64"
 RUN_ALKOS_SCRIPT_GDB_ARGS="-s -S"
-
+RUN_ALKOS_SCRIPT_QEMU_ARGS="-enable-kvm -cpu host"
```

Comment:
does this require updating install dependencies script to install kvm?

Reply:
Install script adjusted